src/legacy/server/saved_objects/service/saved_objects_client.mock.ts

```diff
@@ -17,8 +17,22 @@
  * under the License.
  */
 
-export { SavedObjectsRepository } from './repository';
-export { ScopedSavedObjectsClientProvider } from './scoped_client_provider';
+import { SavedObjectsClientContract } from './saved_objects_client';
+import * as errors from './lib/errors';
 
-import * as errors from './errors';
-export { errors };
+const create = () => {
```

Comment:
This can be simplified to remove the extraneous variable:

```ts
const create = (): jest.Mocked<SavedObjectsClientContract> => ({
  errors,
  create: jest.fn(),
  bulkCreate: jest.fn(),
  delete: jest.fn(),
  bulkGet: jest.fn(),
  find: jest.fn(),
  get: jest.fn(),
  update: jest.fn(),
});
```